mono/metadata/boehm-gc.c

```diff
@@ -945,11 +947,33 @@ mono_gc_free_fixed (void* addr)
 	GC_FREE (addr);
 }
 
+#ifdef HEAP_VALIDATION_FREQUENCY
+static int validate_heap = 0;
+static int counter = 0;
+static int validate_frequency = 100;
+
+void
+mono_gc_set_heap_verifier (gboolean enabled)
+{
+	g_message("heap verifier is now: %d", enabled);
```

Comment:
Debug line?

Reply:
Originally yeah but I found it useful for delineating in the log when the verifier was toggled on vs off. I figured that since this was buried in a define that internal devs would be the only one using this anyway.